mod/wikirate/set/type_plus_right/campaign/participant.rb

```diff
@@ -0,0 +1,27 @@
+
+
+# this is pretending as a pointer card
+def item_names args={}
+  initiative_name = cardname.left
+  # find all participant and add to the pointer card
+  editors = Card.search editor_of: { left: initiative_name }, return: "name"
+  editors += Card.search editor_of: { 
+                          left: {
+                            found_by: "#{initiative_name}+project" 
+                          } 
+                        },
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.